developer-docs-site/docs/tutorials/first-transaction.md

```diff
@@ -1,50 +1,90 @@
 ---
-title: "Your first transaction"
+title: "Your First Transaction"
 slug: "your-first-transaction"
 sidebar_position: 1
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Your first transaction
+# Your First Transaction
 
-This tutorial outlines how to generate, submit, and verify transactions submitted to the Aptos Blockchain. The steps for doing so are:
+This tutorial describes, in the following step-by-step approach, how to generate, submit, and verify transactions submitted to the Aptos Blockchain:
 
-1. Create a representation of an account
-2. Prepare a wrapper around the REST interfaces
-3. Prepare a wrapper around the Faucet interface
-4. Combine them into an application, execute and verify
+1. Create a representation of an account.
+  
+  Each Aptos account has a unique account address. The owner of that account holds the public, private key-pair that maps to the Aptos account address and, in turn, the authentication key stored in that account.  
```

Comment:
Should we also mention that the authentication key can be changed without changing the account address?

Reply:
I think probably we should give that detail in the Accounts document. Here in this tutorial, strictly speaking, the user is not even explicitly creating an account and it might be jumping the gun if we mention changing auth keys. 
I am going to take another thorough look at the Accounts doc also separately.